### PR TITLE
Eris lore changes part 1

### PR DESCRIPTION
**Pull request by Ikmachek**
## About The Pull Request
The start of my crusade to change Eris mentions to not reflect Eris lore, this one changes some design disks and guns, NT voice channel and some other minor stuff.

## Why It's Good For The Game
Because we're Occulus, not Eris. I will not settle for ingame not matching lore.

## Changelog
```changelog
fix: Fixed the manufacturer of some guns/disks and changed NT voice into Mekhane voice
```

### CODE REVIEW

**Comment by bearrrrrrrr** [comment]
Absolutely pogular

File: cev_eris.dme [hunk 1]
```diff
@@ -2799,6 +2799,7 @@
 #include "zzz_modular_syzygy\storytellers\lookout.dm"
 #include "zzz_modular_syzygy\storytellers\mentor.dm"
 #include "zzzz_modular_occulus\code\datums\autolathe\guns.dm"
+#include "zzzz_modular_occulus\code\game\Occulus Lore Fixes.dm"
```
Comment (Michiyamenotehifunana) [inline cev_eris.dme:2802]: ikky how the fuck did you put spaces in a file name like this

why does it work

**Review by Michiyamenotehifunana — CHANGES_REQUESTED** [review]
Filenames should generally have no capitals, and use underscores (_) in place of spaces.

**Review by ShadzFen — APPROVED** [review]
Very cool. Modular lore fixes, non-modular changes to say.dm and communications.dm, obviously.  GoOOodo To GoOOoogo

File: cev_eris.dme (hunk 1 above)
Comment (bearrrrrrrr) [reply cev_eris.dme]: i just saw this
what the fuck
HOW DID YOU DO THIS?? @Ikmachek

**Pull request merged**